onnxscript/rewriter/generic_pattern.py

```diff
@@ -300,7 +300,11 @@ def _match_backward(
             # TODO(rama): Handle constant-pattern
             pattern_pred = pattern_value.producer()
             if pattern_pred is None:
-                # pattern_pred is None means the pattern ends here.
+                # pattern_pred is None means the pattern backward search ends here.
+                result = self._match_values_forward(starting_node, matched, stack, graph_value, pattern_value)
```

Comment:
I fixed the same bug here https://github.com/microsoft/onnxscript/pull/1608 but I inserted a similar call at a different place. I think we can bypass this call if ``len(graph_value.uses()) == 1`` to avoid adding too much time.